pynest/examples/sensitivity_to_perturbation.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+#
+# sensitivity_to_perturbation.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+'''
+Sensitivity to perturbation
+---------------------------
+
+This script simulates a network in two successive trials which
+are identical except for an extra input spike in the second realisation.
+(a small perturbation). The network consists of recurrent, randomly
+connected excitatory and inhibitory neurons. Its activity is driven
```

Comment:
Really? Why? I think it makes sense that the first sentence states what is the main point of the example